integrationtests/src/test/java/org/axonframework/integrationtests/eventhandling/TrackingEventProcessorTest.java

```diff
@@ -788,5 +1114,38 @@ public boolean hasNextAvailable(int timeout, TimeUnit unit) {
         public void close() {
 
         }
+
+
+    }
+    private void waitForSegmentStart(int segmentId) throws InterruptedException {
```

Comment:
I reckoned adding a timeout here would add a lot of complexity. Since they're already defined using an annotation on the test cases themselves, I preferred this way.